exercises/hello-world/hello_world.cpp

```diff
@@ -0,0 +1,16 @@
+#include "hello_world.h"
+
+// Use everything from the 'std' namespace.
+// This lets us write 'string' instead of 'std::string'.
+using namespace std;
+
+namespace hello_world {
+
+// Define the function itself. This could have also been written as 'std::string
+// hello_world::hello()'.
+string hello() {
+    // Return the string we need.
+    return "Hello!";
```

Comment:
As you suggested, `"Change me!"` sounds better.
Alternatives are `"Fix me!"` or `"Start here"`, but `"Change me!"` is direct.

Reply:
Done, I went with "Fix me!"